keyboards/rgbkb/pan/readme.md

```diff
@@ -0,0 +1,19 @@
+# Pan
+
+![pan](https://i.imgur.com/Q6pqCFF.jpg)
+
+Ortho or Staggered DIY kit with RGB
+
+* Keyboard Maintainer: [RGBKB](https://github.com/RGBKB)
+* Hardware Supported: Pan, atmega32a or Proton-C
+* Hardware Availability: [rgbkb.net](https://www.rgbkb.net/), [GitHub](https://github.com/RGBKB/Keyboard-files/tree/master/Pan)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb rgbkb/pan/rev1_atmega -km default
```

Comment:
```suggestion
    make rgbkb/pan/rev1_atmega:default
```

Reply:
Shouldn't we be moving to compile for the QMK CLI?